cmake-modules/FolderList.cmake

```diff
@@ -4,30 +4,31 @@ macro(GetFolderList project)
     if(${project} STREQUAL CERTIFICATES)
         DownloadDepVersion(sdk/core azure-core 1.3.1)
         DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+    elseif(${project} STREQUAL IDENTITY)
+        DownloadDepVersion(sdk/core azure-core 1.3.1)
```

Comment:
Will you sync this version number with VCPKG config at some point?

It will be hard to keep this numbers updated for all services in the future!

Reply:
yes , part of the next ticket https://app.zenhub.com/workspaces/adp-native-team-5f68c995a9966e1e074f078d/issues/azure/azure-sdk-for-cpp/3252